be/db/dev-db/docker-compose.yaml

```diff
@@ -0,0 +1,16 @@
+version: '3.1'
+services:
+  db:
+    image: postgres:latest
```

Comment:
![image](https://user-images.githubusercontent.com/33936483/115154266-a5a24580-a082-11eb-9ee3-f73fdb0dfa80.png)

Reply:
@Orzelius you are right, but don't assume that everybody knows why. Either explain or link to article that explains. 
But yes the reason being if they update docker image it can break the system. Its usually a good idea to use latest when trying out stuff or on your own projects where this does not affect anybody, but you, but for bigger dev projects fixing version will improve stability and will avoid issues from updates. Also fix it to a LTS version if applicable.

Reply:
'postgres:latest' will be changed to 'postgres:13' with commit 6c32dcb59324ecaa4d2920b45c4daab4e59e8c4c.